content/webapp/pages/whats-on.js

```diff
@@ -409,16 +408,20 @@ export class WhatsOnPage extends Component<Props> {
                           v={{ size: 'xl', properties: ['margin-bottom'] }}
                         >
                           {firstExhibition ? (
-                            <Layout12>
-                              <FeaturedCardExhibition
-                                exhibition={firstExhibition}
-                                background={'cream'}
-                                color={'black'}
-                              />
-                            </Layout12>
+                            <div data-test-id="featured-exhibition">
```

Comment:
Should this be added to `FeaturedCardExhibition`?

Reply:
that's probably a better shout, yeah. Thanks